packages/terra-demographics-banner/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "terra-demographics-banner",
+  "main": "lib/DemographicsBanner.js",
+  "private": true,
+  "version": "0.0.0",
+  "description": "terra-demographics-banner",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-ui.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-demographics-banner",
+    "DemographicsBanner",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-ui/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-ui#readme",
+  "devDependencies": {
+    "terra-props-table": "^0.1.0"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
```

Comment:
Looks like I had a code conflict on my branch. I fixed it here. https://github.com/cerner/terra-core/pull/191/commits/de319ec334c14f16107f08769e49696070cdc4d3